File: DSA basic q for beg/digittoword.py
# Program to print the given digit in words

def printvalue(digit):
    if digit=='1':
        print("ONE",end= " ")
    elif digit=='2':
        print("TWO",end= " ")
    elif digit=='3':
        print("THREE",end= " ")
    elif digit=='4':
        print("FOUR",end= " ")
    elif digit=='5':
        print("FIVE",end= " ")
    elif digit=='6':
        print("SIX",end= " ")
    elif digit=='7':
        print("SEVEN",end= " ")
    elif digit=='8':
        print("EIGHT",end= " ")
    elif digit=='9':
        print("NINE",end= " ")
    elif digit=='10':
        print("TEN",end= " ")

def printword(N):
    i = 0
    length = len(N)
    while i<length:
        printvalue(N[i])
        i+=1

N = "678"
print(N,"in word") 
printword(N)

File: DSA basic q for beg/java dsa beginers/Digittoword.java
public class Digittoword {

    void printValue(char digit) {
        switch (digit) {
            case '1':
                System.out.print("one ");
                break;
            case '2':
                System.out.print("two ");
                break;
            case '3':
                System.out.print("three ");
                break;
            case '4':
                System.out.print("four ");
                break;
            case '5':
                System.out.print("five ");
                break;
            case '6':
                System.out.print("six ");
                break;
            case '7':
                System.out.print("seven ");
                break;
            case '8':
                System.out.print("eight ");
                break;
            case '9':
                System.out.print("nine ");
                break;
            case '0':
                System.out.print("zero ");
                break;
            default:
                break;
        }
    }

    void cnvrtToWord(String N) {
        int i = 0;
        int len = N.length();
        while (i < len) {
            printValue(N.charAt(i));
            i += 1;
        }
    }

    public static void main(String[] args) {
        String N = "123";
        Digittoword dtw = new Digittoword();
        System.out.print("the digit : "+N+" in word : ");
        dtw.cnvrtToWord(N);
    }
}
